src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/DataBlocksController.cs

```diff
@@ -34,6 +34,13 @@ public DataBlocksController(IDataBlockService dataBlockService, IReleaseService
                 Ok(await _dataBlockService.CreateAsync(releaseId, dataBlock)));
         }
 
+        [HttpDelete("datablocks/{id}")]
+        public async Task<ActionResult> DeleteDataBlockAsync(DataBlockId id)
+        {
+            await _dataBlockService.DeleteAsync(id);
```

Comment:
I wonder if we should add another response type here, should the delete fail of the deleted file not be found no alternative responses are given.

Reply:
I've added an alternative response now and put in a test for it @jamescutts